xapian-applications/omega/templates/query

```diff
@@ -106,7 +106,8 @@ $if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesiz
 </div>
 </div>
 </td>
-<td><b><a href="$if{$opt{clicklog},$html{$or{$env{SCRIPT_NAME},omega.cgi}?FMT=$url{clicklog}&QUERYID=$url{$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}&DOCID=$url{$id}&URL=$url{$field{url}}},$html{$field{url}}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
+$set{queryid,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}
```

Comment:
This should be done **outside** `$hitlist{}` (it doesn't depend on the document, so we can just calculate it once instead of once per document shown).

Also, it ought to be gated by `$opt{clicklog}` - we don't want to waste effort calculating it at all when click logging isn't enabled.